### PR TITLE
Remove the __weak__ attrs, just let users set AMY_CORES and PCM_SIZE at compile time. Everything else you'd edit `amy_config.h` for.

### PR DESCRIPTION
**Pull request by bwhitman**

### CODE REVIEW

**Comment by bwhitman** [comment]
The `_weak_` stuff was causing more problems (in python, and Arduino) than it was worth

**Comment by bwhitman** [comment]
Now it's like `amy_start(cores, reverb, chorus)`. None of those are evaluated a-rate so shouldn't slow down things

File: examples/AMY_Test_ESP32/AMY_Test_ESP32.ino
Comment (dpwe) [inline examples/AMY_Test_ESP32/AMY_Test_ESP32.ino:1]: If this is supposed to be didactic, maybe name the parameters in comments, e.g. 
```
amy.begin(/* cores= */ 1, /* reverb= */ 0, /* chorus= */ 0);
```

Comment (dpwe) [reply examples/AMY_Test_ESP32/AMY_Test_ESP32.ino:1]: Looks great.  I agree that making these parameters settable at run-time has minimal downside.

**Pull request merged**